### ISSUE TITLE
Cannot install kadira on 1.1.0.2

### ISSUE THREAD

**Issue by rclai** [meteorhacks/kadira#156]
Ubuntu 14.04

```
meteor add meteorhacks:kadira
 => Errors while adding packages:             
                                              
While checking for meteorhacks:kadira-binary-deps@1.2.1:
error: No compatible binary build found for this package. 
Contact the package author and ask them to publish it for your platform.
```

**Comment by arunoda** [comment]
Okay. That's the weird.
Kadira does not have `meteorhacks:kadira-binary-deps` package now.

**Comment by arunoda** [comment]
I tried in both 64 bit and 32 bit.
I don't have any issues.
May be you can remove all the versions of kadira and add it back.

**Comment by rclai** [comment]
How do I do that? Is there some cache somewhere? I already deleted the local folder inside the .meteor folder and it still did the same thing.

**Comment by arunoda** [comment]
try adding `meteorhacks:kadira@2.20.1`

**Comment by rclai** [comment]
That worked. That was really weird.

**Comment by arunoda** [comment]
I guess, this issue was fixed with Meteor 1.1.
